test/src/test/java/hudson/model/LabelLoadStatisticsQueueLengthTest.java

```diff
@@ -116,9 +122,9 @@ public void queueLengthReflectsBuildableItemsAssignedLabel()
         // Verify that the queue length load stat continues to reflect the labels assigned to the items in the queue.
         float labelQueueLengthNew = label.loadStatistics.queueLength
                 .getLatest(TimeScale.SEC10);
-        assertTrue(
-                "After assigning an alternate label to the job, the queue length load statistic for the queued builds should not decrease.",
-                labelQueueLengthNew >= labelQueueLength);
+        assertThat("After assigning an alternate label to the job, the queue length load statistic for the "
+                        + "queued builds should not decrease.",
```

Comment:
Gratuitous formatting change, could have left the message line untouched (and kept the source more readable IMO).